backend/benefit/applications/services/pdf_templates/composed_accepted_private.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-<p>Työllisyydenhoidon Helsinki-lisän myöntäminen työnantajille</p>
+<p>Työllisyydenhoidon Helsinki-lisä myöntäminen työnantajille</p>
```

Comment:
Why was this changed? I think the former version is grammatically correct and the new is incorrect.

Reply:
no idea why, fixed